sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ConnectionString/ConnectionStringParser.cs

```diff
@@ -37,8 +37,9 @@ public static void GetValues(string connectionString, out string instrumentation
             }
             catch (Exception ex)
             {
-                AzureMonitorTraceExporterEventSource.Log.ConnectionStringError(ex);
-                throw new InvalidOperationException("Connection String Error: " + ex.Message, ex);
+                var newEx = new InvalidOperationException("Connection String Error: " + ex.Message, ex);
```

Comment:
Curious - is this `ex.Message` localized or not (or not decided at this moment)?

Reply:
I honestly don't know if the `ex.Message` is ever localized. In theory, exceptions could be thrown from any dependency library so we would have to ask did every library owner localize their messages.
At the moment we're not localizing our own messages.